utils/install.sh

```diff
@@ -396,7 +425,16 @@ install_wasmedge_tensorflow() {
 
 install_image_extensions() {
     if [ $IM_EXT_COMPAT == 1 ]; then
-        get_wasmedge_image_deps
+
+        [ "$EXT_V_SET_WASMEDGE_IM" -eq 0 ] && VERSION_IM=$VERSION &&
+            remote_version_availabilty second-state/WasmEdge-image "$VERSION_IM"
+
+        [ "$EXT_V_SET_WASMEDGE_IM_DEPS" -eq 0 ] && VERSION_IM_DEPS=$VERSION
+
+        [ "$(printf %s\\n%s\\n "$VERSION_IM_DEPS" "0.8.2")" == "$(printf %s\\n%s "$VERSION_IM_DEPS" "0.8.2" | sort --version-sort)" ] &&
+            remote_version_availabilty second-state/WasmEdge-image "$VERSION_IM_DEPS" &&
+            get_wasmedge_image_deps
+
```

Comment:
@q82419 This part checks if the selected version is greater than 0.8.2 then this will not get executed further.

Reply:
Tests:
```
root@3fb60088d61a:/# VERSION_IM_DEPS=0.8.1
root@3fb60088d61a:/#  [ "$(printf %s\\n%s\\n "$VERSION_IM_DEPS" "0.8.2")" == "$(printf %s\\n%s "$VERSION_IM_DEPS" "0.8.2" | sort --version-sort)" ] && echo "install" || echo "don't"
install
root@3fb60088d61a:/# VERSION_IM_DEPS=0.8.2
root@3fb60088d61a:/#  [ "$(printf %s\\n%s\\n "$VERSION_IM_DEPS" "0.8.2")" == "$(printf %s\\n%s "$VERSION_IM_DEPS" "0.8.2" | sort --version-sort)" ] && echo "install" || echo "don't"
install
root@3fb60088d61a:/# VERSION_IM_DEPS=0.9.0-rc.1
root@3fb60088d61a:/#  [ "$(printf %s\\n%s\\n "$VERSION_IM_DEPS" "0.8.2")" == "$(printf %s\\n%s "$VERSION_IM_DEPS" "0.8.2" | sort --version-sort)" ] && echo "install" || echo "don't"
don't
root@3fb60088d61a:/#
```

Reply:
Thanks. I tested that and it really did a great job.